src/pool/singleton/arc.rs

```diff
@@ -81,10 +81,10 @@ use core::{
     sync::atomic,
 };
 
-#[cfg(armv6m)]
+#[cfg(any(armv6m, avr))]
```

Comment:
because `avr` doesn't have atomics, I don't think it will even try to include anything in `pool` so I don't think these changes are needed

Reply:
I think you made these changes because of the incorrect target name in `build.rs`

Reply:
Fixed.